__e2e__/peruse.spec.js

```diff
@@ -242,8 +244,84 @@ describe( 'main window', () =>
         expect( bookmarks ).toMatch( 'shouldappearinbookmarks' );
 
     } );
+    it( 'can check if settings menu exists', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
+
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        const menuExists = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__BUTTON, WAIT_FOR_EXIST_TIMEOUT );
+        expect( menuExists ).toBeTruthy();
+    } );
 
+    it( 'can open settings menu', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
 
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BUTTON );
+        await delay( 4500 );
+        const settings = await client.waitForExist( BROWSER_UI.SETTINGS_MENU, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        expect( settings ).toBeTruthy();
+    } );
+    it('can open settings menu and checks if Bookmarks,History,Toggle exist', async () => 
+    {
+        expect.assertions( 4 );
+        const { client } = app;
+        await delay( 4500 );
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BUTTON );
+        await delay( 4500 );
+        const settings = await client.waitForExist( BROWSER_UI.SETTINGS_MENU, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
+        const bookmarks = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__BOOKMARKS, WAIT_FOR_EXIST_TIMEOUT );
+        await delay( 2500 );
```

Comment:
do we need each of these delays?

Reply:
No we don't, I've tested out and kept the ones that we need.